src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+    private boolean isPrivate;
+
+    public boolean isPrivate() {
```

Comment:
You can do a better job with the generalisation. The constructor can be pushed to this super class along with few more methods.